examples/lighting-app/nrfconnect/main/ZclCallbacks.cpp

```diff
@@ -36,13 +36,13 @@ void MatterPostAttributeChangeCallback(const chip::app::ConcreteAttributePath &
     if (clusterId == OnOff::Id && attributeId == OnOff::Attributes::OnOff::Id)
     {
         ChipLogProgress(Zcl, "Cluster OnOff: attribute OnOff set to %u", *value);
-        LightingMgr().InitiateAction(*value ? LightingManager::ON_ACTION : LightingManager::OFF_ACTION,
-                                     AppEvent::kEventType_Lighting, size, value);
+        GetAppTask().GetLightingDevice().InitiateAction(*value ? PWMDevice::ON_ACTION : PWMDevice::OFF_ACTION,
+                                                        AppEvent::kEventType_Lighting, value);
     }
     else if (clusterId == LevelControl::Id && attributeId == LevelControl::Attributes::CurrentLevel::Id)
```

Comment:
Level control is not part of window covering device type demo
Why this PR impact the lightning demo ?

Reply:
Because I needed to refactor `LightingManager` to make it more generic and non-singleton to avoid boiler plate code related to the PWM handling.